var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -25,7 +25,7 @@ class Slate(Package):
     variant('mpi',    default=True, description='Build with MPI support.')
     variant('openmp', default=True, description='Build with OpenMP support.')
 
-    depends_on('cuda@9:', when='+cuda')
+    depends_on('cuda@9:10.2.89', when='+cuda')
```

Comment:
```suggestion
    depends_on('cuda@9:10', when='+cuda')
```
This might be better if additional CUDA 10.X.Y releases come out and you expect them to work.